xskillscore/core/probabilistic.py

```diff
@@ -844,12 +852,22 @@ def rank_histogram(observations, forecasts, dim=None, member_dim="member"):
         Defaults to None meaning compute over all dimensions
     member_dim : str, optional
         Name of ensemble member dimension. By default, 'member'.
+    random_for_tied : bool
+        Whether assign tied ranks random rank, see Hamill 2001
```

Comment:
Perhaps we could be a bit more explicit?
"Whether to randomly generate ranks for tied values so that, on average, tied values result in a flat histogram - see Hamill 2001"

Reply:
implemented